tests/unit_tests/serialization.cpp

```diff
@@ -1313,15 +1315,15 @@ TEST(Serialization, portability_signed_tx)
   ASSERT_TRUE(epee::string_tools::pod_to_hex(tse.mask) == "796309c7e57439028f111714bd04c8bbe22167bd2f7c04c21dc99b0c16478003");
   // ptx.construction_data.change_dts
   ASSERT_TRUE(tcd.change_dts.amount == 7784000000000);
-  ASSERT_TRUE(cryptonote::get_account_address_as_str(testnet, tcd.change_dts.addr) == "T6T7VwjLn6f5YEkhDeXKN6BPAyqNEyT1RQGXJohw9D4JerRqYSFLUFKULt2sfMGv8nBnUDRhVx5GbBLWeMQt8HeS1ZK4PVYEU");
+  ASSERT_TRUE(cryptonote::get_account_address_as_str(testnet, tcd.change_dts.addr) == "F8ER6NJ6zka6keUKJjX8ry44mVaXuQeVg5dPsuW3gyWRDzxCXpwuHVkMCNmrXZEMVHMFo5zEkoNTeb95hkqWgzMDSWFvana");
```

Comment:
Maybe, it will be better to use some defines or variables to prevent copying of same addresses.